README.md

```diff
@@ -1,62 +1,55 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Bookstore: Use Redux in React components.
 
-> Description the project.
+Get started by copying and pasting the following in your terminal:
 
+- 1
 
-## Built With
+```
 
-- Major languages
-- Frameworks
-- Technologies used
+```
 
-## Live Demo (if available)
+- 2
```

Comment:
- [ ] It will be very nice to have details of the project and how to clone it like : (repo link)